src/api/app/views/webui/user/edit.html.haml

```diff
@@ -8,15 +8,16 @@
   = "Editing User Data for User #{@displayed_user.login}"
 
 = form_tag(action: "save") do
-  %p
-    = hidden_field_tag("user", @displayed_user.login)
-    = label_tag(:realname, "Name:")
-    %br
-    = text_field_tag(:realname, @displayed_user.realname)
-  %p
-    = label_tag(:email, "e-Mail:")
-    %br
-    = text_field_tag(:email, @displayed_user.email, required: true, email: true)
+  - unless CONFIG[:ldap_mode]
```

Comment:
[Nitpick] I don't think we should be calling `CONFIG[:ldap_mode]` from the view. Instead we should assign it to an instance variable like `@ldap_mode_enabled`. Very minor point but the view should't know if we stored this config variable in a constant `CONFIG` or if its from an active record model or whatever.